<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fibonacci</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSpinBox::up-button 
{ subcontrol-position:  right ;width:80px;height:60px }

QSpinBox::down-button 
{ subcontrol-position:  left;width:80px;height:60px}


 QSlider::groove:horizontal 
{
     border: 10px solid #999999;
     height:30px;
     background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
     margin: 12px 0;
 }

 QSlider::handle:horizontal {
     background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
     border: 11px solid #5c5c5c;
     width: 50px;
     margin: -20px 0;
     border-radius: 20px;
 }

	QScrollBar:vertical
	{
		  width: 77px;
		  margin: 50px 0 50px 0;
		  border: 1px solid #222222;
	}

	QScrollBar::handle:vertical
	{
		  min-height: 30px;
		  border-radius: 2px;
	}

	QScrollBar::add-line:vertical
	{
		  border: 1px solid #1b1b19;
		  border-radius: 2px;
		  height: 40px;
		  subcontrol-position: bottom;
		  subcontrol-origin: margin;
	}

	QScrollBar::sub-line:vertical
	{
		  border: 1px solid #1b1b19;
		  border-radius: 2px;
		  height: 40px;
		  subcontrol-position: top;
		  subcontrol-origin: margin;
	}

	QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
	{
		  border: 1px solid black;
		  width: 50px;
		  height: 30px;
		  background: white;
	}

	QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
	{
		  height: 40px;
	}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabss">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Next</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Fibonacci values</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99999999</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbNextFib">
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;For fibonacci Value  &lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$v&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;next = value*&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#333333; background-color:#ffffff;&quot;&gt;φ&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-style:italic; color:#333333; background-color:#ffffff;&quot;&gt;next = &lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#333333; background-color:#ffffff;&quot;&gt;$n&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-style:italic; color:#333333; background-color:#ffffff;&quot;&gt;real value = &lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#333333; background-color:#ffffff;&quot;&gt;$r&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; color:#0000ff; background-color:#ffffff;&quot;&gt;where &lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;?&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:600; color:#0000ff; background-color:#ffffff;&quot;&gt;φ&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;=1.618033988749894848204&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>It is,</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Value</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lbFinoIden">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>12</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>DO</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="lbIdenTmp">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>160</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;v1=&lt;span style=&quot; font-weight:600;&quot;&gt;F^2*5+4 &lt;/span&gt;or v2=&lt;span style=&quot; font-weight:600;&quot;&gt; F^2*5-4 &lt;/span&gt;&lt;br /&gt;if v is integer &lt;span style=&quot; font-weight:600;&quot;&gt;F=&amp;gt;&lt;/span&gt;fibonacci &lt;br /&gt;v1=&lt;span style=&quot; font-weight:600;&quot;&gt; %v1 &lt;/span&gt;&lt;br /&gt;vv1=&lt;span style=&quot; font-weight:600;&quot;&gt;%r1&lt;/span&gt;&lt;br /&gt;v2=&lt;span style=&quot; font-weight:600;&quot;&gt; %v2&lt;/span&gt;&lt;br /&gt;vv2=&lt;span style=&quot; font-weight:600;&quot;&gt;%r2&lt;/span&gt;&lt;br /&gt;&lt;span style=&quot;color:#ff0000;&quot;&gt;%r&lt;/span&gt;&lt;br /&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>generate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="title">
        <string>Generate</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>30</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>generate</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextBrowserInteraction</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Golden Ratio</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QSpinBox" name="spinBox">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbReadValue">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbGrValue">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;a name=&quot;formula&quot;&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Verdana,Trebuchet MS,Lucida,Arial,Helvetica,sans-serif'; font-size:x-large; font-weight:600; color:#cc0000;&quot;&gt;B&lt;/span&gt;&lt;span style=&quot; font-family:'Verdana,Trebuchet MS,Lucida,Arial,Helvetica,sans-serif'; font-size:x-large; font-weight:600; color:#cc0000;&quot;&gt;inet's Formula, Fibonacci number&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/img/fibonacci-formula-phi.png&quot; /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MathJax_Math'; font-size:18px; font-style:italic; color:#333333; background-color:#ffffff;&quot;&gt;φ=&lt;/span&gt;&lt;span style=&quot; font-family:'Verdana,Trebuchet MS,Lucida,Arial,Helvetica,sans-serif'; font-size:medium; color:#000000; background-color:#ffffff;&quot;&gt;1·61803 39887 49894 84820 45868 34365 ...&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;http://www.maths.surrey.ac.uk/hosted-sites/R.Knott/Fibonacci/fibFormula.html&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
